src/pages/komodo-defi-framework/api/v20/my_recent_swaps/index.mdx

```diff
@@ -0,0 +1,390 @@
+export const title = "Komodo DeFi Framework Method: My Recent Swaps V2";
+export const description = "The my_recent_swaps method returns the data of the most recent atomic swaps executed by the Komodo DeFi Framework API node.";
+
+# my\_recent\_swaps
+
+**my\_recent\_swaps (from\_uuid page\_number=1 limit=10 my\_coin other\_coin from\_timestamp to\_timestamp)**
+
+The `my_recent_swaps` method returns the data of the most recent atomic swaps executed by the Komodo DeFi Framework API node. Please note that all filters (my\_coin, from\_timestamp, etc.) are combined using logical AND.
+
+## Arguments
+
+| Structure       | Type   | Description                                                                                                                            |
+| --------------- | ------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| filter          | Object | [FilterCriteria](/komodo-defi-framework/api/common_structures/#filter-criteria) containing all the filtering criteria for the swaps    |
+| paging\_options | Object | [PagingOptionsObject](/komodo-defi-framework/api/common_structures/#paging-options) containing all the paging options for the request. |
```

Comment:
filter and paging_options are flattened structs ref. https://github.com/KomodoPlatform/komodo-defi-framework/blob/af571608c34f38d82e3c421f9bd61c5265db0cf8/mm2src/mm2_main/src/lp_swap/swap_v2_rpcs.rs#L373-L379
Which mean the json should look as something like this
```json
{
  "my_coin": "BTC",
  "other_coin": "ETH",
  "from_timestamp": 1622505600,
  "to_timestamp": 1625097600,
  "limit": 10,
  "page_number": 1,
  "from_uuid": "550e8400-e29b-41d4-a716-446655440000"
}
```

Reply:
Is this consistent across all methods? I see `PagingOptionsEnum` is within existing codebase for lightning and some tx_history methods

Reply:
> Is this consistent across all methods?

No, but it should be, I understand your point. For lightning and tx_history it's not flattened. BTW, this flattened structs are the same structs that are used for `my_recent_swaps` legacy/v1 so it's exactly the same request as v1.